Doc/library/pdb.rst

```diff
@@ -75,6 +75,7 @@ The debugger's prompt is ``(Pdb)``, which is the indicator that you are in debug
    arguments of the ``p`` command.
 
 
+.. _pdb-commandline:
```

Comment:
Let's use the more common spelling:
```console
❯ rg "^\.\. _[a-z-]+-command.*line:" Doc | wc -l
       4

❯ rg "^\.\. _[a-z-]+-cli:" Doc | wc -l
      22
```

```suggestion
.. _pdb-cli:
```

Reply:
I went with "commandline" as it's marginally clearer, but fair point on frequency.